azerothcore/azerothcore-wotlk
# fix(Core/Items): Armor type requirement for needing items in LFG grou…

…p comes into play only for items above 70 ilvl.

Fixed #7467



## Issues Addressed:

- Closes #7467
- Closes https://github.com/chromiecraft/chromiecraft/issues/190
- Closes https://github.com/chromiecraft/chromiecraft/issues/1460

## Tests Performed:

- Not tested

## How to Test the Changes:


1. Queue for a dungeon finder for some classic low level dungeon.
2. Loot some items with armor type not fit to your class/level (e.g. cloth item for warrior or mail item for 42 lvl shaman)
3. See you can need on that item.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
